app/scripts/proactive/config.js

```diff
@@ -237,6 +237,8 @@ switch (family) {
     case OperatingSystemFamily.WINDOWS:
         isWindows = true;
         break;
+    case OperatingSystemFamily.MAC:
+        isMac = true;
```

Comment:
please add break here

Reply:
why?

Reply:
or do you want isWindows to be false

Reply:
I don't want to edit the existing code. Adding a break for mac means I have to declare the isWindows variable outside the switch.